docs/book/src/reference/envtest.md

```diff
@@ -25,7 +25,12 @@ err = testEnv.Stop()
 Logs from the test runs are prefixed with `test-env`.
 
 ### Configuring your test control plane
-You can use environment variables and/or flags to specify the `api-server` and `etcd` setup within your integration tests.
+
+The controller-runtime’s [envtest](https://godoc.org/sigs.k8s.io/controller-runtime/pkg/envtest) framework requires `kubectl`, `kube-apiserver`, and `etcd` binaries to simulate the API portions of a real cluster. 
+
+For projects built with `v3+`, these binaries and configuration is done via the `test` makefile target. It will create the `testbin/` directory where the binaries will be downloaded.  
+
+You can use environment variables and/or flags to specify the `kubectl`,`api-server` and `etcd` setup within your integration tests.
```

Comment:
Link needed.

Reply:
Hi @estroz, 

I did not get what link should be added here? This text was not changed at all and it is added because bellow it has the envvar/flags details.